spec/acceptance/windows_spec.rb

```diff
@@ -0,0 +1,35 @@
+# run a test task
+require 'spec_helper_acceptance'
```

Comment:
Should you have any failure cases in here?

Also how do you run these? Asking for a friend...

Reply:
Hi @eoinmcq,

I'll defer these to @tphoney :)